sdk/docs/manually-written/sdk/tutorials/setup-assistant/installation.rst

```diff
@@ -14,7 +14,7 @@ You need to install:
 1. `Visual Studio Code <https://code.visualstudio.com/download>`_.
 2. JDK 11 or greater. If you don't already have a JDK installed, try `Eclipse Adoptium <https://adoptium.net>`_.
 
-   As part of the installation process you may need to set up the ``JAVA_HOME`` variable. You can find instructions for this process on :doc:`Windows,macOS, and Linux here <path-variables>`.
+   As part of the installation process you may need to set up the ``JAVA_HOME`` variable. You can find instructions for this process on :doc:`Windows,macOS, and Linux in the :ref:`path-variables <path-variables>` section.
```

Comment:
I don't think the `ref` is necessary here, since we're already inside of a :doc: directive this probably breaks compilation.